packages/connect-ui/src/components/Loader.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import { createGlobalStyle } from 'styled-components';
+
+const Style = createGlobalStyle`
```

Comment:
Why not simply style the 'loader' `div` below? `createGlobalStyle` is usually used for global styles, don't think it's a good idea to use it for a component.

Reply:
because it takes time and doing it blocks me from delivering other stuff. and as I wrote in the description of this PR - this design is going to change probably so I did nto want to do one work twice

Reply:
I am only writing regarding the use of `createGlobalStyle`. It could be written as

```suggestion
const Style = styled.div`
```

Reply:
indeed, cool, changed